catroid/src/main/java/org/catrobat/catroid/camera/FaceAndTextDetector.kt

```diff
@@ -196,23 +195,22 @@ object FaceAndTextDetector : ImageAnalysis.Analyzer {
         imageHeight: Int
     ) {
         val frontCamera = StageActivity.getActiveCameraManager().isCameraFacingFront
-        val oldAPI = Build.VERSION.SDK_INT < Build.VERSION_CODES.M
         val aspectRatio = imageWidth.toFloat() / imageHeight
         val faceBounds = face.boundingBox
 
         val facePosition = if (ProjectManager.getInstance().isCurrentProjectLandscapeMode) {
             val relativeX = faceBounds.exactCenterY() / imageHeight
             val relativeY = faceBounds.exactCenterX() / imageWidth
             coordinatesFromRelativePosition(
-                if (oldAPI) relativeX else 1 - relativeX,
+                1 - relativeX,
                 SCREEN_WIDTH / aspectRatio,
                 if (frontCamera) relativeY else 1 - relativeY,
                 SCREEN_WIDTH.toFloat()
             )
         } else {
             val relativeX = faceBounds.exactCenterX() / imageHeight
             coordinatesFromRelativePosition(
-                if (frontCamera.xor(oldAPI)) 1 - relativeX else relativeX,
+                if (frontCamera.xor(false)) 1 - relativeX else relativeX,
```

Comment:
```suggestion
                if (frontCamera) 1 - relativeX else relativeX,
```
@ndrnour please also rebase: there have been many changes in the meantime in catroid/src/main/java/org/catrobat/catroid/camera/FaceAndTextDetector.kt because of https://github.com/Catrobat/Catroid/commit/5f865e47086a1a54d32c5da0cee6e996ee08035e and we have no functional automatic tests for these sensors, i.e., must test them manually with real phones (cannot test camera with emulator, cannot mock), and also on different Android versions. When you rebase, we can see the changes in the latest develop version here.

Reply:
done. this branch is up to date with develop branch.